<launch>

<group ns = "leader_ukf">
   <node pkg="ukf" name="ukf_1" type="force_estimatee" output="screen" >
     <param name="topic_imu" value="/firefly1/imu" />
     <param name="topic_mocap" value="/firefly1/ground_truth/odometry" />
     <param name="topic_drone" value="1" />
   </node>
</group>

<group ns = "follower_ukf">
   <node pkg="ukf" name="ukf_2" type="force_estimatee" output="screen" >
     <param name="topic_imu" value="/firefly2/imu" />
     <param name="topic_mocap" value="/firefly2/ground_truth/odometry" />
     <param name="topic_drone" value="2" />
   </node>
</group>

<node name="payload_ukf" pkg="payload_measure" type="so3" output="screen" />

</launch>
